### ISSUE TITLE
`player_box_yyyy.rds` data has only two columns

### ISSUE THREAD

**Issue by bmacGTPM** [sportsdataverse/fastRhockey-data#1]
I just tried downloading `player_box_2022.rds` and noticed it has only two columns, `game_id` and `season`.  So figured I'd mention it, since based on the name, I'm guessing you meant to have more columns!

I chose another one at random (`player_box_2017.rds`) and it also has two columns, so it's not just the current season. 

```
-- NHL Player Boxscore Information from fastRhockey data repository ----------------------------------- fastRhockey 0.3.0 --
i Data updated: 2022-03-24 08:33:30 EDT
# A tibble: 6 x 2
     game_id season
       <int>  <dbl>
1 2021021016   2022
2 2021021018   2022
3 2021021011   2022
4 2021021010   2022
5 2021021012   2022
6 2021021017   2022
```

**Comment by saiemgilani** [comment]
Yeah something has hit the fan and I have been trying to fix it the past couple of days juggling projects. I am hopeful to have it fixed asap. Good to see you Brian!

**Comment by saiemgilani** [comment]
Phew, justttt a typo, should be fixed now. Please confirm it's working on your end and we'll close this issue

**Comment by saiemgilani** [comment]
This repository is no longer the source of data for the SportsDataverse packages. Data is produced by per-sport repos and published as GitHub release assets on https://github.com/sportsdataverse/sportsdataverse-data/releases, which is what the current `hoopR` / `wehoop` / `fastRhockey` / `cfbfastR` loaders and `sportsdataverse-py` read.

NHL player box scores are produced by **fastRhockey-nhl-data** (rebuilt in 2026, including the game_rosters repair) and read by `fastRhockey::load_nhl_player_box()` / `sportsdataverse.nhl`. Closing as superseded; this repo is being archived.
